kubernetes.go

```diff
@@ -321,6 +339,7 @@ func (c *KubernetesClient) Watch() error {
 		}
 
 		if !more {
+			c.stopWatchers()
```

Comment:
In this Watch function, should it be added a defer func() (as in connect function) to stop watchers too?